openedx/features/enterprise_support/README.rst

```diff
@@ -0,0 +1,13 @@
+Enterprise Support
+------------------
+
+This directory contains a Django application to support usage of
+enterprise features within edx-platform. The majority of the capabilities
+are provided through the external edx-enterprise library that can be found
+here: `https://github.com/edx/edx-enterprise`_.
+
+Note: this Django app is only necessary due to limitations in extending
+edx-platform through installed libraries. The eventual goal is to have
+all of this functionality provided through edx-enterprise. New code should
```

Comment:
@haikuginger thanks, that's helpful. IMO it would still be the case that this client would live in a separate repo, and it would be pip installed into edx-platform. We are trying to get to the point where edx-platform does not need to be modified in order to plug in new capabilities.

Shall I just remove this paragraph for now, and then you can add in any details you'd like to see here?